enarxbot.py

```diff
@@ -4,6 +4,123 @@
 import github
 import os
 
+ORGANIZATION_LABELS = {
+    'amd sev': {
+        'name': 'amd sev',
+        'description': 'Issues related to AMD SEV',
+        'color': '000000'
+    },
```

Comment:
```suggestion
    'amd sev': {
        'rename': ['sev'],
        'description': 'Issues related to AMD SEV',
        'color': '000000'
    },
```

I think the name field is redundant and can be removed. Conversely, we should add the renames here. This is so there is no separation between related parts of the configuration.